### ISSUE TITLE
Segfault in bun

### ISSUE THREAD

**Issue by jerebtw** [oven-sh/bun#15315]
### How can we reproduce the crash?

I used `bunx --bun shadcn@latest add` and wanted to add these components: badge, breadcrumb, button, calendar, card, dialog, drawer, dropdown-menu, form, input, label, pagination, popover, scroll-area, select, separator, sheet, sidebar, skeleton, slider, sonner, switch, table, tabs, toast

And after sheet.tsx bun crashed

```
✔ Checking registry.
✔ Updating tailwind.config.ts
✔ Updating src\app\globals.css
✔ Installing dependencies.
√ The file badge.tsx already exists. Would you like to overwrite? ... yes
√ The file breadcrumb.tsx already exists. Would you like to overwrite? ... yes
√ The file button.tsx already exists. Would you like to overwrite? ... yes
√ The file calendar.tsx already exists. Would you like to overwrite? ... yes
√ The file card.tsx already exists. Would you like to overwrite? ... yes
√ The file dialog.tsx already exists. Would you like to overwrite? ... yes
√ The file drawer.tsx already exists. Would you like to overwrite? ... yes
√ The file dropdown-menu.tsx already exists. Would you like to overwrite? ... yes
√ The file form.tsx already exists. Would you like to overwrite? ... yes
√ The file label.tsx already exists. Would you like to overwrite? ... yes
√ The file input.tsx already exists. Would you like to overwrite? ... yes
√ The file popover.tsx already exists. Would you like to overwrite? ... yes
√ The file scroll-area.tsx already exists. Would you like to overwrite? ... yes
√ The file select.tsx already exists. Would you like to overwrite? ... yes
√ The file separator.tsx already exists. Would you like to overwrite? ... yes
√ The file sheet.tsx already exists. Would you like to overwrite? ... yes
```

### Relevant log output

```shell
Bun v1.1.34 (5e5e7c60) Windows x64
Windows v.win10_fe
CPU: sse42 avx avx2 avx512
Args: "bunx" "--bun" "shadcn@latest" "add"
Features: Bun.stderr Bun.stdin(2) Bun.stdout fetch(83) jsc spawn(2) transpiler_cache(14) tsconfig(2) 
Builtins: "bun:main" "node:assert" "node:buffer" "node:child_process" "node:constants" "node:crypto" "node:events" "node:fs" "node:fs/promises" "node:http" "node:https" "node:net" "node:os" "node:path" "node:process" "node:readline" "node:stream" "node:string_decoder" "node:timers/promises" "node:tls" "node:tty" "node:url" "node:util" "node:util/types" "node-fetch" 
Elapsed: 59665ms | User: 937ms | Sys: 234ms
RSS: 0.35GB | Peak: 0.35GB | Commit: 0.45GB | Faults: 96685

panic(main thread): Segmentation fault at address 0x5
oh no: Bun has crashed. This indicates a bug in Bun, not your code.
```

### Stack Trace (bun.report)

Bun v1.1.34 ([`5e5e7c6`](<https://github.com/oven-sh/bun/tree/5e5e7c60f1004c79a3f87e1f5470ee19d57f10d0>)) on windows x86_64 [BunxCommand]

Segmentation fault at address 0x00000005

- *5 unknown/js code*
- `llint_entry`

Features: Bun.stderr, Bun.stdin, Bun.stdout, dotenv, fetch, jsc, spawn, transpiler\_cache, tsconfig

<sub>Sentry Issue: <strong><a href="https://bun-p9.sentry.io/issues/5610708776/">BUN-226</a></strong></sub>

**Comment by jerebtw** [comment]
This is strange, I am using the latest version of Bun

```bash
❯ bun --version
1.1.36
```

**Comment by heimskr** [comment]
Fixed by #15357.
